src/api/renderer.c

```diff
@@ -203,12 +203,28 @@ static int f_font_gc(lua_State *L) {
 }
 
 
+static RenTab checktab(lua_State *L, int idx) {
+  RenTab tab = {.enabled = false, .offset = 0};
+  if (lua_isnoneornil(L, idx)) {
+    return tab;
+  }
+  luaL_checktype(L, idx, LUA_TTABLE);
+  lua_getfield(L, idx, "tab_offset");
+  if (lua_isnoneornil(L, -1)) {
+    return tab;
+  }
+  tab.offset = luaL_checknumber(L, -1);
```

Comment:
In theory, doesn't it mean that tab.offset == 0 equals disabled? Is there an edge case I am missing?

Reply:
No, it just means start calculating the stops as if the draw started from 0, so encountering a tab will still advance to the next stop.
When disabled it advances of `space_advance * tab_size`.